### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by darylrobbins** [ivarprudnikov/atom-language-groovy#1]
Store package settings files in the `settings/` directory instead of `scoped-properties/`

```
Package.loadSettings (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/package.js:584:9)
<unknown> (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/package.js:184:43)
Package.measure (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/package.js:163:15)
Package.load (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/package.js:177:12)
PackageManager.loadPackage (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/package-manager.js:355:14)
PackageManager.activatePackage (/opt/homebrew-cask/Caskroom/atom/latest/Atom.app/Contents/Resources/app.asar/src/package-manager.js:434:30)
```

**Comment by ivarprudnikov** [comment]
Committed my changes, but at the moment having some issues with package updates in `apm`. Will continue on this issue a bit later.

**Comment by ivarprudnikov** [comment]
Issue resolved in v0.3.0
